### PR TITLE
fix: set default export format in project

### PR DESCRIPTION
**Pull request by mydiemho**
export wasn't doing anything because there wasn't a default format.

AB#17437

### CODE REVIEW

File: src/redux/actions/projectActions.ts [hunk 1]
```diff
@@ -1,26 +1,38 @@
...
+
     closeProject(): void;
+
     exportProject(project: IProject): Promise<void> | Promise<IExportResults>;
+
     loadAssets(project: IProject): Promise<IAsset[]>;
+
     loadAssetMetadata(project: IProject, asset: IAsset): Promise<IAssetMetadata>;
+
     saveAssetMetadata(project: IProject, assetMetadata: IAssetMetadata): Promise<IAssetMetadata>;
```
Comment (elizabethhalper) [inline src/redux/actions/projectActions.ts]: nit: Any reason for adding the extra spaces? I think when exporting an interface it is more readable and consistent with our other formatting to leave them out.

File: src/redux/actions/projectActions.test.ts [hunk 2]
```diff
@@ -76,14 +76,41 @@ describe("Project Redux Actions", () => {
...
-
         const result = await projectActions.saveProject(project)(store.dispatch, store.getState);
 
         expect(result.version).toEqual(appInfo.version);
     });
 
+    it("Save Project action on new project correctly add default export format", async () => {
+        projectServiceMock.prototype.save = jest.fn((project) => Promise.resolve(project));
+
+        const skeletonProject = MockFactory.createTestProject("TestProject");
```
Comment (elizabethhalper) [inline src/redux/actions/projectActions.test.ts]: nit: I would call this either "project" or "testProject" like we have been doing in the rest of the tests

Comment (mydiemho) [reply src/redux/actions/projectActions.test.ts]: project is below, this one is use to create the test project

File: src/redux/actions/projectActions.ts (hunk 1 above)
Comment (mydiemho) [reply src/redux/actions/projectActions.ts]: oh autoformat on webstorm did that, I can undo it

File: src/models/applicationState.ts [hunk 3]
```diff
@@ -44,6 +44,7 @@ export enum ErrorCode {
     ProjectUploadError = "projectUploadError",
     ProjectDeleteError = "projectDeleteError",
     SecurityTokenNotFound = "securityTokenNotFound",
+    ExportFormatNotFound = "exportFormatNotFound",
```
Comment (wbreza) [inline src/models/applicationState.ts]: When specifying a new error code you will also need to setup the localized strings for the error code.

Comment (mydiemho) [reply src/models/applicationState.ts]: fixed

**Pull request merged**